data/sles4sap/angi_hana_cluster.conf

```diff
@@ -0,0 +1,48 @@
+primitive rsc_SAPHanaTpg_%SID%_HDB%HDB_INSTANCE% ocf:suse:SAPHanaTopology \
```

Comment:
Is this file the "angi version" of file `os-autoinst-distri-opensuse/data/sles4sap/hana_cluster.conf` ?
If it is so may we aligne the two a little bit more, for example the Topology primitive in this file is named `rsc_SAPHanaTpg_%SID%_HDB%HDB_INSTANCE%` and the other use similar but not same `rsc_SAPHanaTopology_%SID%_HDB%HDB_INSTANCE%`   (Tpg vs Topology). I do not only propose to edit this new file but maybe also fix the other.

Reply:
It's based on the official HA/SAP guys documentation, they are really to the 3 letter naming, So I left it as it.

Reply:
Even better, as you can see, the configuration file includes the name of the module, so if you need exact name of the resource or something really universal, you could parse it from the  **`crm configure show`** command.

Reply:
we can do alignment in os-autoinst-distri-opensuse/data/sles4sap/hana_cluster.conf and later in a next PR